functests/utils/images/images.go

```diff
@@ -7,53 +7,55 @@ import (
 	gomega "github.com/onsi/gomega"
 )
 
-var registry string
-var images map[string]imageLocation
+var (
+	registry string
+	tag string
+	images map[string]imageLocation
+)
 
 const (
 	// TestUtils is the image name to be used to retrieve the test utils image
 	TestUtils = "testutils"
-	// SctpTester is the image name to be used to retrieve the sctptester image
-	SctpTester = "sctptester"
 	// Dpdk is the image name to be used to retrieve the dpdk image
 	Dpdk = "dpdk"
 )
 
 func init() {
 	registry = os.Getenv("IMAGE_REGISTRY")
+	tag = os.Getenv("IMAGE_TAG")
 
 	images = map[string]imageLocation{
-		SctpTester: {
-			name:    "sctptester",
-			registy: "quay.io/openshift-kni/",
-			version: "4.5",
-		},
 		TestUtils: {
-			name:    "cnftest-utils",
+			name:    "cnf-tests",
```

Comment:
In order to achieve maximum flexibility, I think it would make sense to have both the registry AND the image+tag configurable.
For this reason, I think it's better to have one env variable for the registry, and one with imagename+tag for each test image:

Something like:
- `IMAGE_REGISTRY` defaulting to `quay.io/openshift-kni/`
- `DPDK_TEST_IMAGE` defaulting to `dpdk:4.5`
- `CNF_TEST_IMAGE` defaulting to `cnf-tests:4.5`